lib/clearance/session.rb

```diff
@@ -54,7 +54,9 @@ def sign_in(user, &block)
       status = run_sign_in_stack
 
       if status.success?
-        cookies[remember_token_cookie] = user && user.remember_token
+        cookies[remember_token_cookie] = cookie_options.merge(
+          value: user && user.remember_token,
+        )
```

Comment:
This merge ensures that we are maintaining the custom domain configuration that may have been passed when the user logs in. Without this merge, `sign_in` and `add_cookie_to_headers` end up creating 2 different cookies, one with the custom domain, and one without.